pages/linux/grub-install.md

```diff
@@ -0,0 +1,12 @@
+# grub-install
```

Comment:
You can add an example with `--modules` because is useful is some scenarios when at boot time you get `device not found`:
```
grub-install ... --modules="{{part_gpt part_msdos}}"
```

Reply:
Can you suggest a change? I've never used that option so I don't feel I have the authority to know what context it can be used in.

Reply:
cc @navarroaxel

Reply:
cc @navarroaxel

Reply:
The example was added as the last example of this page.